src/Components/ProgressBar.jsx

```diff
@@ -20,17 +27,21 @@ const ProgressBar = ({ currentSectionName, totalSections }) => {
       sectionName = "Terms of Offence";
       currentSection = 3;
       break;
-    case "conv-1" || "circ-1" || "ter-1":
+    case "eligible":
```

Comment:
What do you think about creating enums for these raw strings? For example in the calculatorTypes file, we could add:
```
export enum SectionName {
     ter = "Terms Of Offense",
     circ = "Surrounding Circumstances"
}
```

And then we can reference them like so: `SectionName.ter`, `SectionName.circ` etc, just to reduce the amount of raw strings we use :)

Reply:
Great Point! Done!